src/site/content/en/blog/web-bundles/index.md

```diff
@@ -166,6 +166,27 @@ To try out a Web Bundle:
 
 Everything magically works.
 
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/web-bundles/preact-todomvc.mp4" 
+            type="video/mp4; codecs=h264">
+  </video>
+  <figcaption class="w-figcaption">
+    The Preact implementation of TodoMVC working offline as a web bundle
+  </figcaption>
+</figure>
```

Comment:
@robdodson I can't figure out why this video isn't working.

HTML seems OK: https://web.dev/handbook/markup-media/#video

Do videos **have** to include a webm version? That's the only thing I omitted.

Video URL seems OK: https://storage.googleapis.com/web-dev-assets/web-bundles/preact-todomvc.mp4

Reply:
hm, changing the type attribute to: `type="video/mp4"` instead of `type="video/mp4"; codecs=h264` fixed it for me.

Interestingly, if I leave codecs=h264 in, the video _does_ work in Safari. Maybe we should just remove that codecs field from all of our videos?